Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -239,10 +239,11 @@ namespace WhiteBox
     {
         if (m_renderMesh.has_value())
         {
-            (*m_renderMesh)->UpdateMaterial(m_material);
+            WhiteBoxMaterial material = m_material;
+            material.m_visible = true;
```

Comment:
question: is it necessary to make a copy of material here? can we not just modify `m_material` (even if that means having to write back to `material.m_visible`)?

Reply:
yea, I guess we need both boolean one for the material and another to be overridden separately? How I think this should work is you flag the whitebox mesh to be invisible but if you are editing it then it should become visible again right?